src/components/Tooltip.js

```diff
@@ -24,22 +24,24 @@ class Tooltip {
    * }
    */
   constructor({
-    parent, title, items, position, unxkcdify,
+    parent, title, items, position, unxkcdify, legendColor, strokeColor,
   }) {
     this.title = title;
     this.items = items;
     this.position = position;
     this.filter = !unxkcdify ? 'url(#xkcdify)' : null;
+    this.legendColor = legendColor;
+    this.strokeColor = strokeColor;
 
     this.svg = parent.append('svg')
       .attr('x', this._getUpLeftX())
       .attr('y', this._getUpLeftY())
       .style('visibility', 'hidden');
 
     this.tipBackground = this.svg.append('rect')
-      .style('fill', 'white')
-      .attr('fill-opacity', 0.85)
-      .attr('stroke', '#aaa')
+      .style('fill', this.legendColor)
+      .attr('fill-opacity', .9)
+      .attr('stroke', this.strokeColor)
```

Comment:
border of the tooltip better be not exactly the same as legend. But not sure what is a good way